### PR TITLE
Update documentation to include information about SUSE Linux support

### PR DESCRIPTION
**Pull request by juanmanuelug**
## Description
|Related issue|
|---|
|[#3771](https://github.com/wazuh/wazuh-documentation/issues/3771)|

Hi team,

With the inclusion of a dedicated SUSE Linux support in the Vulnerability Detector module, Wazuh's documentation needs to be updated to include references about this operating system in the correspondent pages.

The main points to cover include:

- [x] The Vulnerability Detector section must include the new feeds and the SUSE operating systems as supported platforms.

- [x] The Vulnerability Detector reference has to include the new provider as well.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

**Review by alromeros — APPROVED** [review]
LGTM

File: source/getting-started/use_cases/vulnerability_detection.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ In order to detect vulnerable software, Wazuh uses a database of Common Vulnerab
 - `CVEs for Red Hat and CentOS Linux distributions <https://access.redhat.com>`_
 - `CVEs for Debian Linux distributions <https://www.debian.org>`_
 - `CVEs for Arch Linux distributions <https://security.archlinux.org>`_
+- `CVEs for SUSE Linux distributions <https://ftp.suse.com>`_
```
Comment (chemamartinez) [inline source/getting-started/use_cases/vulnerability_detection.rst:16]: Please replace this URL to https://www.suse.com/support/security/oval/

File: source/learning-wazuh/vuln-detection.rst [hunk 2]
```diff
@@ -111,6 +111,16 @@ In the ``/var/ossec/etc/ossec.conf`` file of the Wazuh manager, scroll down to t
         <enabled>no</enabled>
         <update_interval>1h</update_interval>
       </provider>
+      <provider name="suse">
+        <enabled>no</enabled>
+        <os>15-server</os>
```
Comment (chemamartinez) [inline source/learning-wazuh/vuln-detection.rst:116]: We will finally reverse this list to sort them from older to newer versions as the rest of providers. Please, replace them by:
```xml
      <os>11-server</os>
      <os>11-desktop</os>
      <os>12-server</os>
      <os>12-desktop</os>
      <os>15-server</os>
      <os>15-desktop</os>
```

File: source/user-manual/capabilities/vulnerability-detection/how_it_works.rst [hunk 3]
```diff
@@ -15,6 +15,7 @@ The global vulnerability database is created automatically, currently pulling da
 - `<https://security.archlinux.org>`_: Used to pull CVEs for Arch Linux distributions.
 - `<https://nvd.nist.gov/>`_: Used to pull CVEs from the National Vulnerability Database.
 - `<https://feed.wazuh.com/>`_: Used to pull the MSU feed with CVEs and patches for Microsoft products.
+- `<https://ftp.suse.com>`_: Used to pull CVEs for SUSE Linux Enterprise distributions.
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/how_it_works.rst:18]: The same, replace the URL.

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 4]
```diff
@@ -307,6 +325,18 @@ The following configuration will update the vulnerability database for Ubuntu, D
           <update_interval>1h</update_interval>
         </provider>
 
+        <!-- SUSE Linux Enterprise OS vulnerabilities -->
+        <provider name="suse">
+          <enabled>yes</enabled>
+          <os>15-server</os>
```
Comment (chemamartinez) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:331]: Reverse this list please:
```xml
      <os>11-server</os>
      <os>11-desktop</os>
      <os>12-server</os>
      <os>12-desktop</os>
      <os>15-server</os>
      <os>15-desktop</os>
```

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 5]
```diff
@@ -261,6 +262,58 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, configu
...
+| SLES 12        | `<https://ftp.suse.com/pub/projects/security/oval/suse.linux.enterprise.server.12.xml>`_ |
++----------------+------------------------------------------------------------------------------------------+
+| SLED 12        | `<https://ftp.suse.com/pub/projects/security/oval/suse.linux.enterprise.desktop.12.xml>`_|
++----------------+------------------------------------------------------------------------------------------+
+| SLES 15        | `<https://ftp.suse.com/pub/projects/security/oval/suse.linux.enterprise.server.15.xml>`_ |
++----------------+------------------------------------------------------------------------------------------+
+| SLED 15        | `<https://ftp.suse.com/pub/projects/security/oval/suse.linux.enterprise.desktop.15.xml>`_|
++----------------+------------------------------------------------------------------------------------------+
+
+And then placed accordingly in the custom location:
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:287]: Change this sentence please:
```
Feeds can be fetched from a custom repository using the `url` attribute for each supported OS, as shown in this example:
```

File: source/user-manual/capabilities/vulnerability-detection/offline_update.rst [hunk 6]
```diff
@@ -261,6 +262,58 @@ Alternatively, the feeds can be loaded from a local path. To achieve it, configu
...
+        <os url="http://local_repo/suse.linux.enterprise.server.11.xml">11-server</os>
+        <os url="http://local_repo/suse.linux.enterprise.desktop.11.xml">11-desktop</os>
+        <os url="http://local_repo/suse.linux.enterprise.server.12.xml">12-server</os>
+        <os url="http://local_repo/suse.linux.enterprise.desktop.12.xml">12-desktop</os>
+        <os url="http://local_repo/suse.linux.enterprise.server.15.xml">15-server</os>
+        <os url="http://local_repo/suse.linux.enterprise.desktop.15.xml">15-desktop</os>
+        <update_interval>1h</update_interval>
+    </provider>
+
+Alternatively, the feeds can be loaded from a local path with the ``path`` attribute, as shown in this example:
```
Comment (chemamartinez) [inline source/user-manual/capabilities/vulnerability-detection/offline_update.rst:302]:
```
Alternatively, they also can be loaded from a local path as follows:
```

**Review by chemamartinez — APPROVED** [review]
LGTM

**Comment by chemamartinez** [comment]
It will surely be in conflict with https://github.com/wazuh/wazuh-documentation/pull/3796. When the documentation for the Amazon Linux support is merged, we have to resolve conflicts before closing it.

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**